### ISSUE TITLE
Using "to" in accessor leads to SyntaxError: Property name expected after dot (char 10) 

### ISSUE THREAD

**Issue by f1nality** [josdejong/mathjs#2027]
When trying to parse string "elements.to.value" i'm getting:
SyntaxError: Property name expected after dot (char 10)

mathjs: "7.5.0

i've found that there are 
 const NAMED_DELIMITERS = {
    mod: true,
    to: true,
    in: true,
    and: true,
    xor: true,
    or: true,
    not: true
  }

But when using it after "." it doesn't make sense that "to" parsed as delimeter, but not accessor.
Looks like a wrong behavior, is it possible to disabled some named delimeters until issue is resolved?

using elements["to"] instead of elements.to is not a good solution for us since we are making no-code solution and it can be hard for users to understand such notation and I can't transform it underhood since even parse is not working.

**Comment by josdejong** [comment]
Thanks fro reporting Denis, that should indeed just work. I guess indeed the `to` is interpreted as a named delimiter (for converting units). 

As a workaround you can use `elements["to"].value` instead of `elements.to.value`.

Anyone interested in fixing this issue? Help would be welcome.

**Comment by dvd101x** [comment]
I found today the issue with `in` mentioned by Denis.

I was working with temperatures like so.

```py
T = {in:10 degC, out:12 degC}
T.out           # yields 12 degC
T.in            # yields SyntaxError: Property name expected after dot (char 3)
T["in"]         # yields 10 degC
```
This is not new information, just stating that I encountered this example and this thread was useful for working around that bug.

**Comment by gwhitney** [comment]
Just to be concrete here: the proposed, and approved, resolution for this issue report would be for `Obj.to` and `Obj.in` to both parse successfully as ordinary property accesses of properties `to` resp. `in` of the given `Obj`?

**Comment by josdejong** [comment]
Yes. I've fixed this issue now via 8ecf3265fc855953c6638eb6ef9fa863fab3e97f.

**Comment by josdejong** [comment]
Fixed now in `v11.11.2`.
